compiler/z/codegen/OMRCodeGenerator.cpp

```diff
@@ -715,14 +715,24 @@ OMR::Z::CodeGenerator::CodeGenerator()
    self()->setSupportsBCDToDFPReduction();
    self()->setSupportsIntDFPConversions();
 
+   // traps are causing some problems on z/OS and not providing much if any perf benefit
```

Comment:
Could you please provide a better description of the problems you encountered which lead to the decision to disable traps on z/OS? It will help a future reader understand _why_ traps are disabled.